runtime/doc/filetype.txt

```diff
@@ -576,15 +576,15 @@ g:no_man_maps		Disable local mappings for man buffers.
 To use Neovim as a man pager put this in your environment: >
 	export MANPAGER='nvim +Man!'
 
-Note that man will pre-format the text using groff.  Therefore Neovim will
-inevitably display the manual page as it was passed to it from stdin:
-hard-wrapped.  If you prefer soft-wrapping, you can try setting MANWIDTH to a
-very large number, e.g.: >
+Note that man will pre-format the text using groff.  Therefore the MANPAGER
```

Comment:
Could you elaborate on these changes? It's not clear to me what the misunderstanding was.

Reply:
I wrote that it was needed to `set wrap`, a mistake. I also thought that `g:man_hardwrap` controlled the width at which to wrap the man page. When I realized my mistake I decided to provided a clearer explanation.